src/lev-report/ReportFilters.js

```diff
@@ -36,6 +38,13 @@ const ReportFilters = props =>
     }}>
       Usage to
     </DatePicker>
+    <ListFilter hint="Default: All Departments" input={{
+        name: 'currentGroup',
+        groups: props.groups,
```

Comment:
Should `groups` be a part of `input` or should it be its own thing?

I'd think it should be renamed to `options` to be a bit less specific as well.

Reply:
Refactored to put groups into component props as opposed to input